pom.xml

```diff
@@ -77,6 +77,7 @@
 		<jetbrains.annotations.version>23.0.0</jetbrains.annotations.version>
 		<log4j2.version>2.17.2</log4j2.version>
 		<testcontainer.version>1.17.3</testcontainer.version>
+		<urlfrontier.version>2.2</urlfrontier.version>
```

Comment:
this has nothing to do here. None of the module specific dependencies should be specified in the parent pom

Reply:
I thought that is the parent pom. Which one would have been the right one?

Reply:
it is, which is why there shouldn't be anything about urlfrontier there ;-)